service/models.py

```diff
@@ -100,6 +100,7 @@ class Order(db.Model):
     cust_id  = db.Column(db.Integer)             #Customer ID for the order
     order_items = db.relationship('OrderItem', backref='order', lazy = True, \
         cascade = "all,delete") #Items in the order
+    status = db.Column(db.String(80)) # status of the order
```

Comment:
I think it is better to have an Enum type here.

Reply:
Hi @sandeep-nyu, I tried using Enum first, but I think Enum is not serializable as Prof also mentioned it, so it will cause some bugs.

Reply:
ok. 

I think we can convert from Enum to String while serializing, similar to that done in "lab-flask-rest" 
 discussed in class

https://github.com/nyu-devops/lab-flask-rest/blob/e639401686f0b80d45fc00d0aaae273f13461903/service/models.py#L106

https://github.com/nyu-devops/lab-flask-rest/blob/e639401686f0b80d45fc00d0aaae273f13461903/service/models.py#L48

Reply:
@sandeep-nyu Ok got it! Will try